projects/npm-tools/packages/changelog-generator/src/yarn.js

```diff
@@ -24,9 +24,9 @@ function yarn(command) {
 
 		if (status !== 0) {
 			// eslint-disable-next-line no-console
-			error(
+			log(
```

Comment:
Was this intentional to go from error to log?

Reply:
yeah, I changed this intentionally. There were to objects called "error": 
```
const {error} = require('./console');

const {error} = child_process.spawnSync('yarn', [command], {stdio: 'inherit'});
```

So I decided to change the console message to a `log` instead than an error, I think it is not a bug deal...